script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -80,43 +80,43 @@ Even though the sun has just set, it is as quiet as if it were midnight.
 This is the place where I came to check on the noisy dog the night I came back to the mansion.
 <Page24>
 "?"
-Just now, there was something in front of me, a black <thing|lump of meat>.
+Right now, right in front of me, is something resembling a black lump of meat.
 <Page25>
 "――――――"
-My consciousness froze.
+My consciousness freezes.
 Why are those here, I wonder.
-It's obvious, or so I've come to realize.
+It's obvious, I realize.
 <Page26>
 It wasn't just Arcueid who was spotted in the alleyway.
 The person that was with Arcueid would also become a target for capture.
-Didn't I block my thoughts so that I wouldn't be aware of that fact?
+Didn't I block out my thoughts so that I wouldn't be aware of that fact?
 <Page27>
 As if to make up for the delay, I take out my knife.
 At the edge of the street... I step back, keeping my eyes on the corpses that emerged from the thicket.
 At that moment,
 <Page28>
 ――――That's strange.
-Suddenly, the whole area turns into a bright red.
+Suddenly, the whole area is turned a bright red.
 <Page29>
 I hear a rumbling sound.
 It's coming from somewhere very close.
-My whole body is convulsing in extreme pain like a spring.
+My entire body leaps in pain like a released spring.
 I'm thrashing around again and again.
 My consciousness, my body, my vision, the world,
-are all spinning like an emergency light.
+are all spinning like an emergency light.//
```

Comment:
These things don't seem to have a proper name, at least that I could find. Is there a more succinct phrase for these?

Reply:
maybe "siren lights"? I'm ok with emergency lights, google doesn't turn up any real name for them other than "flashers" which sucks.